resources/views/Widgets/Header/NavigationWidget.json

```diff
@@ -108,6 +108,13 @@
                 "name": "Widget.navigationCompanyLogoUrlLabel",
                 "tooltipText": "Widget.navigationCompanyLogoUrlTooltip"
             }
+        },
+        "spacing":{
+            "type": "vertical",
+            "options": {
+                "name": "Widget.widgetSpacing"
+            },
+            "children": "@include(spacing)"
```

Comment:
Ist es hier vielleicht sinnvoller, nur vertikale Abstände konfigurierbar zu machen? In dem fall würde ich einfach ein "spacingVertical" und ein "spacingHorizontal" als globales json ergänzen.

Reply:
Beim TopBar und Breadcrumb Widget, haben die Optionen dazu geführt, dass der Widget editier Button nicht mehr im ShopBuilder war und man somit nicht die Einstellungen öffnen konnte. Beim Navigation Widget haben die Horizontalen margins und paddings ganz gut funktioniert.